com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -1074,6 +1081,7 @@ public unsafe void Deserialize(FastBufferReader reader)
                 if (Header.IsReparented)
                 {
                     reader.ReadValue(out IsLatestParentSet);
+                    reader.ReadValue(out ChildWorldPositionStays);
```

Comment:
nit: Feels wrong injecting this between "IsLatestParentSet" and "latestParent" - those seem a pair so should be next to each other... same on the write side of things, obviously.